File: Parking_backend/app/Models/usuario.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class usuario extends Model
{
    use HasFactory;

    protected $table= 'usuario';

    protected $primaryKey= 'id_usuario';

    protected $fillable = [
        'id_usuario',
        'nombre',
        'a_paterno',
        'a_materno',
        'id_ipn',
        'ruta_identificacion',
        'ruta_fotografia',
        'numero_contacto'
    ]; 

    public $timestamps = false;

    public function cuenta()
    {
        return $this->hasMany(cuenta::class, 'id_cuenta');
    }

    public function visita()
    {
        return $this->hasMany(Visita::class, 'id_visita');
    }
}

File: Parking/app/src/main/java/mx/ipn/escom/bautistas/parking/model/Usuario.kt
package mx.ipn.escom.bautistas.parking.model

import com.google.gson.annotations.SerializedName
import kotlinx.serialization.Serializable

@Serializable
data class Usuario (
    @SerializedName("id_usuario")
    val idUsuario: Long,
    val nombre: String,
    @SerializedName("a_paterno")
    val aPaterno: String,
    @SerializedName("a_materno")
    val aMaterno: String,
    @SerializedName("id_ipn")
    val idIpn: String,
    @SerializedName("ruta_identificacion")
    val rutaIdentificacion: String,
    @SerializedName("ruta_fotografia")
    val rutaFotografia: String,
    @SerializedName("numero_contacto")
    val numeroContacto: String
)
